app/api/v2/views/users.py

```diff
@@ -75,32 +87,34 @@ def post(self):
 
 # checks if a user with the username exists
         # user=ListDatabase.get_user_by_username(username)
-        cursor.execute("SELECT * from users WHERE username=%s AND password=%s,(username, password)")
-        users=cursor.fetchone()
+        cursor.execute("SELECT * from users WHERE username=%s AND password=%s ,(username, password)")
+        users = cursor.fetchone()
         
-        access_token=create_access_token(identity=username)
+        access_token = create_access_token(identity=username)
         return jsonify(access_token=access_token), 200
         
         if not user:
-            invalidUserErrorMsg={
+            invalidUserErrorMsg = {
```

Comment:
This variable is not necessary just jsonify your return message

Reply:
Hello Boniface,

Thank you for this feedback,  Let me try to put it into effect.

Regards
Steven Ennis